lib/modules/manager/pep621/processors/uv.ts

```diff
@@ -175,6 +180,40 @@ export class UvProcessor implements PyProjectProcessor {
   }
 }
 
+function applyGitSource(
+  dep: PackageDependency,
+  depSource: z.infer<typeof UvGitSource>,
```

Comment:
```suggestion
  depSource: UvGitSource,
```

Please no inline infers.